modules/running-compliance-scans-worker-node.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+[id="running-compliance-scans-worker-node_{context}"]
+=  Scheduling the result server pod on a worker node
+
+The result server pod is used to mount the persistent volume (PV) storage volume that stores the raw Asset Reporting Format (ARF) scan results. The `nodeSelector` and `tolerations` attributes allow you to configure the location of the result server pod.
+
+This is helpful for those environments where control plane nodes are not permitted to mount persistent volumes.
+
+.Procedure
+. Create a `ScanSetting` custom resource (CR) for the Compliance Operator:
+.. Define the following `ScanSetting` CR, and save the YAML file, for example, `rs-workers.yaml`:
++
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+kind: ScanSetting
+metadata:
+  name: rs-on-workers
+  namespace: openshift-compliance
+rawResultStorage:
+  nodeSelector:
+    node-role.kubernetes.io/worker: "" <1>
+  pvAccessModes:
+  - ReadWriteOnce 
+  rotation: 3
+  size: 1Gi
+  tolerations:
+  - operator: Exists <2>
+roles:
+- worker
+- master
+scanTolerations:
+  - operator: Exists
+schedule: 0 1 * * *
+----
+<1> The Compliance Operator uses this node to store scan results in ARF format.
+<2> The result server pod tolerates all taints.
+
+
+.. Create the `ScanSetting` CR by running the following command:
```

Comment:
```suggestion
.. Create the `ScanSetting` CR, run the following command:
```

> From our [contributing to docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output)
If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter".